### PR TITLE
Update smarthome.cfg

### PR DESCRIPTION
**Pull request by kgoderis**
Partially fixes https://github.com/openhab/openhab-distro/issues/1037

Signed-off-by: Karel.goderis@me.com

### CODE REVIEW

File: launch/app/runtime/conf/smarthome.cfg [hunk 1]
```diff
@@ -1,9 +1,9 @@
-org.eclipse.smarthome.folder:items=items
-org.eclipse.smarthome.folder:sitemaps=sitemap
-org.eclipse.smarthome.folder:rules=rules
-org.eclipse.smarthome.folder:scripts=script
-org.eclipse.smarthome.folder:persistence=persist
-org.eclipse.smarthome.folder:things=things
+org.openhab.core.folder:items=items
```
Comment (kaikreuzer) [inline launch/app/runtime/conf/smarthome.cfg]: This actually shows me that there was an error in the refactoring. The namespace for all configs has to be `org.openhab.<configname>`, so the `core` is too much in there.
@cweitkamp Can you fix this in the core?

Comment (cweitkamp) [reply launch/app/runtime/conf/smarthome.cfg]: Yes, I will do that. And update #1030 accordingly.

**Comment by kaikreuzer** [comment]
Changes are included in https://github.com/openhab/openhab-distro/pull/1030.

File: launch/app/runtime/conf/smarthome.cfg (hunk 1 above)
Comment (cweitkamp) [reply launch/app/runtime/conf/smarthome.cfg]: Done. See https://github.com/openhab/openhab-core/pull/1314.

**Pull request closed without merging**